### PR TITLE
[TASK] #102605 - TSFE->fe_user removed

### PR DESCRIPTION
**Pull request by brotkrueml**
Resolves: https://github.com/TYPO3-Documentation/Changelog-To-Doc/issues/747
Releases: main

### CODE REVIEW

File: Documentation/ApiOverview/TSFE/Index.rst [hunk 1]
```diff
@@ -166,9 +166,12 @@ Get the language of the current page as integer:
...
 --------------------------------
 
+..  versionchanged:: 13.0
+    The variable has been removed with TYPO3 v13.
+
 ..  code-block:: php
 
-    // !!! discouraged
+    // !!! Not available anymore since TYPO3 v13
     $feUser = $GLOBALS['TSFE']->fe_user;
```
Comment (linawolf) [inline Documentation/ApiOverview/TSFE/Index.rst:175]: I would not leave a wrong example in here. I would remove the code-block and instead link to the migration section

Comment (brotkrueml) [reply Documentation/ApiOverview/TSFE/Index.rst:175]: Personally, I would like to remove the whole part. It was discouraged in the past, and therefore it should not be used.

Comment (linawolf) [reply Documentation/ApiOverview/TSFE/Index.rst:175]: Maybe keep a link to the migration section in v13 and remove in v14? I think use of this was once wide-spread so devs will have to migrate old projects?

Comment (brotkrueml) [reply Documentation/ApiOverview/TSFE/Index.rst]: Done, added a link to the migration.

**Review by linawolf — APPROVED** [review]

**Pull request merged**